template/docs/conf.py.jinja

```diff
@@ -37,6 +37,8 @@ try:
     import sciline.sphinxext.domain_types  # noqa: F401
 
     extensions.append('sciline.sphinxext.domain_types')
+    # See https://github.com/tox-dev/sphinx-autodoc-typehints/issues/457
+    suppress_warnings = ["config.cache"]
```

Comment:
Wait why is this in this block? Shouldn't this be set for all the builds?

Are we using autodoc only for sciline?

Reply:
This suppression is specifically for our custom extension `sciline.sphinxext.domain_types` (bundled with Sciline). It is not needed when we don't use the extension. The try-except is about enabling the extension.